main.py

```diff
@@ -1,10 +1,41 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+    for player, player_info in players.items():
+        race, _ = Race.objects.get_or_create(
+            name=player_info["race"].get("name"),
+            description=player_info["race"].get("description")
+        )
+
+        for skill in player_info["race"].get("skills"):
```

Comment:
Ensure that 'skills' is always a list in the JSON data. If 'skills' is None or not a list, this loop will raise an error. Consider adding a check or default value to handle such cases.